wire-ios/Wire-iOS/Sources/AppDelegate.swift

```diff
@@ -76,6 +77,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     // MARK: - Public Set Property
     var window: UIWindow?
+    var currentAuthorizationFlow: OIDExternalUserAgentSession?
```

Comment:
Is the authorization flow for the whole device or is it per user?

Reply:
per user